### DISCUSSION TITLE
[OP Stack Deployment] Which documentation should I use as existing one is not recommended by OP Stack itself

### DISCUSSION THREAD

**Discussion by OpenArmand** [ethereum-optimism/developers#675] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

i was going to use this to build a chain on testnet:https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup

but it says "Please do not rely on the content of this page as it is currently undergoing maintenance. Code samples and solutions may not function as expected. Please check back for an update or signup to help us revise this page. We welcome your contribution! ❤️"

so what should i do instead? should i wait for the update if it's in like the next few days? or how should i get this tutorial otherwise?

### Additional Information

I’m building OpenWork (https://openwork.technology/), a decentralized work protocol that we plan to build directly on the OP Stack. 

OpenWork enables two strangers to safely enter work contracts, pay in crypto, use escrow services, resolve disputes, build reputations, and maintain self-custody of their profiles, all without the need for a centralized authority of any kind.

Our vision includes a “1 job = 1 block” model, where every completed job is recorded as its own block. By doing this, we aim to make posting, completion, skill verification, and even dispute resolution fully transparent and trustless on-chain. We see the OP Stack as the perfect foundation for creating more openness, fairness, and efficiency in work arrangements.

Optimistic about to working with OP Stack with your support!

**Comment by hash1go** [comment]
Hello.
As far as I know, that warning notice has been there for several months. Therefore, it's unlikely that the notice will be removed and the documentation will be updated in the next few days.
If you encounter any issues with the tutorial, you can ask questions in this forum.

**Reply by OpenArmand** [reply]
thanks! Is it largely correct? Would appreciate if I know in advance what in it is updated or incorrect somehow

**Reply by hash1go** [reply]
I don't know which parts of the documentation are outdated.

**Comment by OpenArmand** [comment]
ok, I have done the dirtywork of figuring this out and can clean up your
documentation. Could there be a small reward for helping? As we've made our
own L2 and have observed gaps and we can patch for you documentation wise.

**Reply by hash1go** [reply]
Are you suggesting improvements to the documentation? You can open an issue or create a pull request.
There is no compensation for documentation contributions. While you may be eligible for grants in future Retro Funding rounds, documentation contributions are out of scope for at least the 2025 Retro Funding.

**Comment by Chomtana** [comment]
Should use this document for contract deployment instead...
https://docs.optimism.io/operators/chain-operators/tools/op-deployer
